docker/taxii/Pipfile

```
@@ -0,0 +1,14 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+bs4 = "*"
```

Use `beautifulsoup4`. `bs4` is a place holder which just imports `beautifulsoup4`.
```suggestion
beautifulsoup4 = "*"
```